### PR TITLE
Proteomics Metadata (and accidently update on WIP_proteinQuant_SIL.md)

### PR DESCRIPTION
**Pull request by Stortebecker**
Hi.

I added the `index.md` file for the Proteomics folder, which I had overlooked before. I also updated the `Proteomics.yml`.

Accidently, I updated WIP_proteinQuant_SIL.md in my master (instead of a separate branch), so this PR includes an update for this tutorial as well. I hope that this does not disturb too much.

Cheers, Florian

### CODE REVIEW

**Comment by Stortebecker** [comment]
Ah, can you please check, if the relative paths in the `requirements` sections of the `Proteomics.yml` are correct? I was not sure about this.

File: metadata/Proteomics.yml [hunk 1]
```diff
@@ -32,16 +32,16 @@ material:
...
+    - "You should always include a contaminant database."
+    - "It may be wise to include mycoplasma databases."
+    - "Depening on your search engine, you need to include a decoy database."
 
     requirements:
     -
-        title: "Galaxy introduction"
-        type: "internal"
-        link: "/Introduction/"
+        title:
```
Comment (bebatut) [inline metadata/Proteomics.yml]: If there is no requirement, please remove the `title`, `type` and `link`

File: metadata/Proteomics.yml [hunk 2]
```diff
@@ -56,42 +56,81 @@ material:
...
-    - "..."
+    - "LC-MS/MS raw files have to be converted before using GalaxyP on most GalaxyP servers."
+    - "SearchGUI can be used for running several peptide search engines at once."
+    - "PeptideShaker can be used to combine and evaluate the results, and to perform protein inference."
+
+    requirements:
+    -
+        title: "Protein FASTA Database Handling"
+        type: "internal"
+        link: "/Proteomics/tutorials/"
```
Comment (bebatut) [inline metadata/Proteomics.yml]: You need to give the name of the file with the tutorial here

File: Proteomics/tutorials/WIP_proteinQuant_SIL.md [hunk 3]
```diff
@@ -5,9 +5,16 @@ tutorial_name: proteinQuant_SIL
 ---
```
Comment (bebatut) [inline Proteomics/tutorials/WIP_proteinQuant_SIL.md]: The `topic_name` is proteomics and `tutorial_name` the same name as the filename

File: Proteomics/tutorials/WIP_proteinQuant_SIL.md [hunk 4]
```diff
@@ -20,16 +27,14 @@ This tutorial deals with protein quantitation via stable isotope labelling (SIL)
...
+The tool settings need to be carefully tested and evaluated manually to obtain optimal results. We will explain this in the section [Evaluation and Optimization of Quantitation Results](#evaluation-and-optimization-of-quantitation-results). 
 
 > ### :pencil2: Hands-on: MS1 Feature Detection
 > 
-> Here, we will use a quantitative comparison of **HEK _OR_ E.coli** cell lysate as a test dataset. In this experiment, the very same cell lysate was once labelled with light, once with heavy **dimethyl _OR_ SILAC** and both samples were subsequently mixed in a certain ratio. Your objective in this hands-on-tutorial is to find out the correct mixing ratio. *And no, the ratio is not mentioned in the description of the dataset on PRIDE. We left out that particular information on purpose. Haha.*
-> 
-> 
+> Here, we will use a quantitative comparison of **HEK _OR_ E.coli** cell lysate as a test dataset. In this experiment, the very same cell lysate was once labelled with light, once with heavy **dimethyl _OR_ SILAC** and both samples were subsequently mixed in a certain ratio.
```
Comment (bebatut) [inline Proteomics/tutorials/WIP_proteinQuant_SIL.md]: It is better in an hands-on part, to give clear instruction. You can take inspiration from the [RNA-seq tutorial](https://galaxyproject.github.io/training-material//RNA-Seq/tutorials/ref_based)

Comment (bebatut) [inline Proteomics/tutorials/WIP_proteinQuant_SIL.md]: An hands-on part ends with 

```
> 
{: .hands_on}
```

File: metadata/Proteomics.yml (hunk 1 above)
Comment (Stortebecker) [inline metadata/Proteomics.yml]: So I never give the "Galaxy Introduction" as a requirement?

File: Proteomics/tutorials/WIP_proteinQuant_SIL.md (hunk 4 above)
Comment (Stortebecker) [inline Proteomics/tutorials/WIP_proteinQuant_SIL.md]: And I liked this part so much...

But can I keep the sentence about the objective? "Your objective in this hands-on-tutorial is to find out the correct mixing ratio."

File: metadata/Proteomics.yml (hunk 1 above)
Comment (bebatut) [inline metadata/Proteomics.yml]: You can but here you removed it :)

**Comment by bgruening** [comment]
Thanks Flo!

**Pull request merged**